pages/common/pdftocairo.md

```diff
@@ -0,0 +1,28 @@
+# pdftocairo
+
+> Converts PDF files to PNG/JPEG/TIFF/PDF/PS/EPS/SVG using cairo.
+> More information: <https://poppler.freedesktop.org>.
+
+- Convert a PDF file to JPEG:
+
+`pdftocairo {{path/to/file}}.pdf -jpeg`
```

Comment:
I think `{{path/to/file}}.pdf` should be replaced with `{{path/to/file.pdf}}` so the syntax can be properly highlighted.

Reply:
Thanks for your feedback, will be making the changes soon.
meanwhile when do I use this {{path/to/file}}.pdf then?

Reply:
I think it's best to be avoided. As far as I've seen so far, arguments are either completely inside the brackets or completely outside them.